course/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 4.1.7 on 2023-04-17 16:01
+# Generated by Django 4.2 on 2023-04-19 23:39
```

Comment:
Please do not manually edit or re-create migrations that already been posted,  generate a new one instead